config/config.go

```diff
@@ -153,6 +166,20 @@ func Validate() error {
 		return errors.New("SSL requires both key and certificate when enabled")
 	}
 
+	// validate heartbeat, notification, and monitoring service endpoints
```

Comment:
For notification and service hearbeat endpoints lets do the following
1) If no endpoint is given then lets ignore this functionality
2) If an endpoint is given and its invalid then lets stop the daemon. 
Essentially we are saying that if a URL is specified then it must be a valid one, no URL is treated as disabling the functionality

For monitoring endpoint the validity check should be based on if monitoring is enabled or not.

Reply:
fixed the issue. please review